symengine/printers/latex.cpp

```diff
@@ -230,6 +232,55 @@ void LatexPrinter::bvisit(const StrictLessThan &x)
     str_ = s.str();
 }
 
+std::string latex(const DenseMatrix &m, const int max_rows, const int max_cols)
```

Comment:
Let's do unsigned
```suggestion
std::string latex(const DenseMatrix &m, const unsigned max_rows, const unsigned max_cols)
```

Can you also fix the formatting as mentioned in https://github.com/symengine/symengine/pull/1847/checks?check_run_id=3721380879 ?